catkit2/services/oceanoptics_spectrometer/oceanoptics_spectrometer.py

```diff
@@ -0,0 +1,187 @@
+'''
+This module contains a service for Ocean Optics Sectrometer
+
+This service is a wrapper around the python seabreeze package
+It provides a simple interface to control the spectro and acquire spectras.
+'''
+
+import threading
+
+from seabreeze.spectrometers import Spectrometer as spct
+from seabreeze.spectrometers import SeaBreezeError
+
+from catkit2.testbed.service import Service
+
+
+class OceanOpticsSpectrometer(Service):
+    '''
+    Service for Ocean Optics Spectrometer.
+
+    This service is a wrapper around the python seabreeze package
+    It provides a simple interface to control the spectro and acquire spectras.
```

Comment:
```suggestion
    It provides a simple interface to control the spectrometer and acquire spectra.
```

Note how "spectra" is the plural form of "spectrum". You'll want to change this everywhere in this file. Also, I think it would be helpful to always write the full word "spectrometer", to avoid any confusion.